Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -1003,7 +1018,7 @@ void EmbedMultipleConfs(ROMol &mol, INT_VECT &res, unsigned int numConfs,
            "does not work with molecules that have multiple fragments. The "
            "boundsMat will be ignored."
         << std::endl;
-    coordMap = nullptr;
+    coordMap = nullptr; //FIXME not directly related to ETKDG, but here I think it should be params.boundsMat = nullptr
```

Comment:
you're correct; nice catch!
Would you fix it please?

Reply:
As we discussed I couldn't resolve this :(